decidim-dev/lib/decidim/dev/test/rspec_support/feature.rb

```diff
@@ -105,6 +105,18 @@ def serialize
     resource.template = "decidim/dummy_resource/linked_dummys"
   end
 
+  feature.register_stat :dummies_count_high, primary: true, priority: Decidim::StatsRegistry::HIGH_PRIORITY do |features, _start_at, _end_at|
+    features.count * 10
+  end
+
+  feature.register_stat :dummies_count_medium, primary: true, priority: Decidim::StatsRegistry::MEDIUM_PRIORITY do |features, _start_at, _end_at|
+    features.count * 100
+  end
+
+  feature.register_stat :dummies_count_low do |features, _start_at, _end_at|
+    features.count * 200
+  end
```

Comment:
Unused?

Reply:
Oh yes, looks like. I'll try to fix it this weekend, otherwise @josepjaume will have to take a look at it

Reply:
I can do it, @mrcasals!